cmd/coordinated/metrics.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2015-2017 Diffeo, Inc.
+// This software is released under an MIT/X11 open source license.
+
+package main
+
+import (
+	"github.com/diffeo/go-coordinate/coordinate"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var coordinateSummary = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Namespace: "diffeo",
+		Subsystem: "coordinate",
+		Name:      "coordinate_summary",
+		Help:      "Summary of coordinate work specs",
+	},
+	[]string{
+		"namespace",
+		"work_spec",
+		"status",
+	},
+)
+
+func init() {
+	prometheus.MustRegister(coordinateSummary)
+}
+
+func observe(coord coordinate.Coordinate) {
```

Comment:
The overall style in the rest of `./cmd/coordinated` is to have public functions and I'd make this one be public too.  That also means writing a docstring.